lib/matplotlib/tests/test_dates.py

```diff
@@ -2,17 +2,25 @@
 import tempfile
 from unittest.mock import Mock
 
-import dateutil
+import dateutil.tz
+import dateutil.rrule
 import numpy as np
 import pytest
-import pytz
 
 from matplotlib.testing.decorators import image_comparison
 import matplotlib.pyplot as plt
 from matplotlib.cbook import MatplotlibDeprecationWarning
 import matplotlib.dates as mdates
 
 
+def __has_pytz():
```

Comment:
double underscore seems a bit overkill

Reply:
I have no strong preference here. I think I used double underscore because I intended it to be ephemeral. I think maybe I should switch it to:

```python
def __has_pytz():
    ...
HAS_PYTZ = __has_pytz()
del __has_pytz()
```

We can drop the `del` line and switch to a single underscore if that's preferred style.